api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -230,36 +225,23 @@
     String playgroundDir = "workspaces_playground/" + workspacePath;
```

Comment:
remove the "must be kept in sync" comment just above here

Reply:
Done - nice catch.